docs/userguide.md

```diff
@@ -69,6 +69,18 @@ factoryResource.use { factory =>
 ## Extensions
 Extensions are packages that monitor some functionality for you - without you having to do anything.
 
+### Http4s
+Http4s package (`datadog-scala-http4s`) provides implementation of [MetricsOps](https://http4s.org/v0.20/api/org/http4s/metrics/metricsops) that is used by [http4s](https://http4s.org) to report both client and server metrics.
```

Comment:
just a nitpick, i kinda like using the `reference style links` (see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) for links that are long - i think it makes for cleaner markdown so i would prefer using that

Reply:
done